awesome_owl/static/src/card/card.js

```diff
@@ -0,0 +1,22 @@
+import { Component, useState} from"@odoo/owl";
```

Comment:
```suggestion
import { Component, useState} from "@odoo/owl";
```
Please focus on having better styling and readability